website/data/docs-nav-data.json

```diff
@@ -1000,6 +1000,10 @@
             "title": "Redshift",
             "path": "secrets/databases/redshift"
           },
+          {
+            "title": "Redis",
+            "path": "secrets/databases/redis"
+          },
```

Comment:
I think it's in alphabetical order so Redis would be before Redshift

Reply:
👍 Will update